modules/checking-deployment-yaml-files.adoc

```diff
@@ -8,13 +8,22 @@
 .Procedure
 
 * Run the following command to check the build-time and deploy-time violations of your security policies in YAML deployment files:
-//TODO: Add link to security policies section
 +
-[source,terminal,subs="+quotes"]
+[source,terminal]
 ----
-$ roxctl deployment check --file=_<yaml_filename>_
+$ roxctl deployment check --file=<yaml_filename> \// <1>
+--namespace=<cluster_namespace> \// <2>
+--cluster=<cluster_name_or_id> \// <3>
+--verbose <4>
 ----
 +
+--
+<1> For the `<yaml_filename>`, specify the YAML file with one or more deployments to send to Central for policy evaluation. You can specify multiple YAML files to send to Central for policy evaluation.
+<2> For the `<cluster_namespace>`, specify a namespace to enhance deployments with context information such as network policies, role-based access controls (RBACs) and services for deployments that do not have a namespace in their specification. The namespace defined in the specification is not changed. The default value is `default`.
+<3> For the `<cluster_name_or_id>`, specify the cluster name or ID that you want to use as the context for the evaluation to enable extended deployments with cluster-specific information.
```

Comment:
This is the name or ID of the cluster in ACS. I don't know whether that's something we need to explicitly mention, as Kubernetes clusters can be named themselves, right?
See ACS Web UI -> Platform Configuration -> Clusters for the name. (e.g. https://stagingdb.demo.stackrox.com/main/clusters)

Reply:
Thank you for your suggestion and I greatly appreciate your feedback, but I think it is appropriate to use the current text as I used the same text that is included in the output of the "roxctl deployment check -h" command:

[agantony@agantony-thinkpadt14sgen2i Downloads]$ ./roxctl deployment check -h
Check deployments for deploy time policy violations, and exit with an non-zero code if at least one of the violated policies has deploy time enforcement turned on.

Usage:
  ./roxctl deployment check [flags]

Flags:
  -c, --categories strings                optional comma separated list of policy categories to run.  Defaults to all policy categories.
      --cluster string                    cluster name or ID to use as context for evaluation. Setting cluster enables enhancing deployments with cluster-specific information.
[...]